internal/packetfabric/cloud_router_connection_test.go

```diff
@@ -229,30 +242,38 @@ func buildMockCloudRouterReadResp(description string) []byte {
 		"speed": "1Gbps",
 		"state": "Requested",
 		"cloud_circuit_id": "PF-AP-LAX1-1002",
-		"account_uuid": "a2115890-ed02-4795-a6dd-c485bec3529c",
+		"account_uuid": "%s",
```

Comment:
The byte slice returned by this function is not processed further by any `fmt` functions. The final value in this place will be literally `%s`.